File: School.Domain/Entities/Enrollment.cs
using System;
using System.Collections.Generic;

namespace School.Domain.Entities
{
    public class Enrollment
    {
        public int Id { get; set; }
        public DateTime DateOfEnrollment { get; set; }
        public int UnitId { get; set; }
        public Unit Unit { get; set; }
        public int ContactId { get; set; }
        public Contact Contact { get; set; }
        public int AddressId { get; set; }
        public Address Address { get; set; }
        public List<Discipline> Disciplines { get; set; }
    }
}

File: School.WebApi/Views/Default/src/app/organization/Entities/Enrollment.ts
import { Address } from "./Address";
import { Contact } from "./Contact";
import { Discipline } from "./Discipline";
import { Unit } from "./Unit";

export class Enrollment {

     id:number;
     dateOfEnrollment:Date;
     unitid:number;
     unit:Unit;
     contactid:number;
     contact:Contact;
     addressid:number;
     address:Address;
     disciplines: Discipline[];
}